core/templates/dev/head/components/fuzzy_rule_editor_directive.html

```diff
@@ -0,0 +1,2 @@
+<input type="hidden"/>
+<button type="button" class="btn btn-danger" ng-click="removeEntry()">Remove this input</button>
```

Comment:
Yup it does not trigger.

I have used input of type hidden because then select2 overlaps with `Remove this input` button. I think it has something to do with `select2Node` being firstChile of directive element.